manifests/init.pp

```diff
@@ -394,6 +396,7 @@
   Boolean $wipe_db_on_cookie_change                                                                = false,
   String $cluster_partition_handling                                                               = 'ignore',
   Variant[Integer[-1],Enum['unlimited'],Pattern[/^(infinity|\d+(:(infinity|\d+))?)$/]] $file_limit = 16384,
+  Optional[Integer] $oom_score_adj                                                                 = 0,
```

Comment:
if the default is 0, you cannot set it to undef so `Optional` doesn't make sense here. I'm not sure if the default here should be undef or 0 :thinking:

Reply:
You're right, but since the current code write a value in ```templates/default.erb```, I will remove the ```Optional``` and keep the default value :)